pastis/plotting.py

```diff
@@ -21,7 +21,7 @@
 from pastis.config import CONFIG_PASTIS
 from pastis.e2e_simulators.luvoir_imaging import LuvoirAPLC
 import pastis.e2e_simulators.webbpsf_imaging as webbpsf_imaging
-import pastis.util
+import pastis.util as util
```

Comment:
I would actually prefer if we keep this to simply `import pastis.util`. The reason is that many packages have a `util` module and importing them without an alias makes it easier to understand what is going on, and to avoid conflicts.

Also, since the only thing that's added in this script is the plotting of the mu map, which we don't have yet, lets' revert all changes in this script as well (see comment for `pastis_analysis.py`)